webrender/src/device/query_gl.rs

```diff
@@ -71,16 +71,18 @@ pub struct GpuFrameProfile<T> {
     samplers: QuerySet<GpuSampler<T>>,
     frame_id: FrameId,
     inside_frame: bool,
+    ext_debug_marker: bool
```

Comment:
github shows something off with indentation here

Reply:
I don't see anything wrong in my editor.

![screenshot 2018-07-30_08-35-16](https://user-images.githubusercontent.com/26073376/43398058-ed173b72-93d4-11e8-9b12-6fc46403e08c.png)